packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -1,3 +1,4 @@
+import { CertificateAuthority } from '@aws-cdk/aws-acmpca';
```

Comment:
Same here - no unqualified imports outside your module.

```suggestion
import * as acmpca from '@aws-cdk/aws-acmpca';
```